tests/test_sinogram.py

```diff
@@ -135,4 +139,84 @@ def test_multidim(num_ang):
     _nrms = np.sqrt(np.mean((s - reference_sinograms) ** 2, axis=-1)) / np.linalg.norm(
         reference_sinograms, axis=-1
     )
-    np.testing.assert_array_less(_nrms, SK_TOL, "Error in image projections.")
+    np.testing.assert_array_less(
+        _nrms, SK_TOL_FORWARDPROJECT, "Error in image projections."
+    )
+
+
+def test_back_project_single(masked_image, num_ang):
+    """
+    Test Line.backproject on a single stack of line projections or sinogram. Compares the reconstructed image to original.
```

Comment:
This description seems off. Doesn't look like we're comparing to the original image.

Reply:
<img width="494" alt="Screenshot 2024-08-06 at 12 12 58 PM" src="https://github.com/user-attachments/assets/14a0c396-e62b-4418-941e-bb5091294174">

changed the description, good catch